tests/SideBySide/ConnectAsync.cs

```diff
@@ -178,6 +179,102 @@ public ConnectAsync(DatabaseFixture database)
 			}
 		}
 
+		[Fact]
```

Comment:
Should probably add a `SecondaryDbFactAttribute` in `Attributes.cs` and use this with the Secondary Database facts.  That way users can skip Secondary Database if they do not have one readily configured and do not have Root permissions to create one.